app/backend/src/clj/teet/thk/thk_export.clj

```diff
@@ -81,9 +81,9 @@
 
                  ;; TEET id for phase and activity
                  "phase_teetid"
-                 (str (:db/id lifecycle))
+                 (str (thk-mapping/uuid->number (:integration/id lifecycle)))
```

Comment:
Are these nil safe after the change / do they need to be ?

Reply:
It is required that all things sent in this integration have that id.

It could be better checked and reported if not, but we want the export to fail if they aren't present.